arviz/plots/jointplot.py

```diff
@@ -80,26 +82,22 @@ def jointplot(trace, varnames=None, figsize=None, textsize=None, kind='scatter',
         axjoin.scatter(x, y, **joint_kwargs)
     elif kind == 'hexbin':
         if gridsize == 'auto':
-            gridsize = int(len(trace)**0.35)
+            gridsize = int(len(x)**0.5)
```

Comment:
did you change 0.35 to 0.5 on purpose?

Reply:
oops, was playing around with that number and forgot to change back. let me look again to see if it *should be* on purpose

Reply:
Here it is with 0.35, 0.42, and 0.5, respectively.  I've got no strong feelings, so I'll change back to 0.5!

![image](https://user-images.githubusercontent.com/2295568/42851872-e517c6ae-89fb-11e8-8064-b226f4dfe0bf.png)

![image](https://user-images.githubusercontent.com/2295568/42851877-ea83c160-89fb-11e8-9dbb-8e0bd9ae7b53.png)

![image](https://user-images.githubusercontent.com/2295568/42851882-efc4b242-89fb-11e8-8e53-ee7c3c097e33.png)